meta-xt-prod-cockpit-rcar-driver-domain/recipes-kernel/kernel-module-cldatdrv/kernel-module-cldatadrv_git.bb

```diff
@@ -1,4 +1,4 @@
-SUMMARY = "CPURTT drive Module"
+SUMMARY = "RCAR cluster data drive Module"
 LICENSE = "CLOSED"
```

Comment:
Could you please squash this commit with the previous one ('cluster_drv: introduce the driver ...').